src/Core/Layer/Layer.js

```diff
@@ -175,4 +182,74 @@ const ImageryLayers = {
     },
 };
 
+/**
+ * Interface implemented by {@link GeometryLayer} instances capable of displaying color layers
+ * @interface
+ */
+/**
+ * Add a color layer to display on this geometry
+ *
+ * @function
+ * @name CanDisplayColorLayer#addColorLayer
```

Comment:
what is `CanDisplayColorLayer`?
where is documentation?

Reply:
`CanDisplayColorLayer` *is* the documentation (see jsdoc virtual comments http://usejsdoc.org/tags-interface.html#virtual-comments)

Reply:
yes thanks,
but `{@link CanDisplayColorLayer}` links on nothing

Reply:
Oh right. This is fixed now.